File: 프로그래머스/0/120922. 종이 자르기/종이 자르기.py
def solution(M, N):
    answer = 0
    cnt_m = 0
    cnt_n = 0
    while M != 1:
        cnt_m += 1
        M -= 1
    while N != 1:
        cnt_n += 1
        N -= 1
    answer = cnt_m + (cnt_m + 1) * cnt_n
    return answer

File: 프로그래머스/0/120922. 종이 자르기/종이 자르기.java
class Solution {
    public int solution(int M, int N) {
        int cnt_m = 0;
        int cnt_n = 0;
        while (M > 1) {
            cnt_m++;
            M--;
        }
        while (N > 1) {
            cnt_n++;
            N--;
        }
        int answer = cnt_m + (cnt_m + 1) * cnt_n;
        return answer;
    }
}